.github/workflows/tf_apply_staging.yml

```diff
@@ -0,0 +1,94 @@
+name: "Terraform apply staging"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "terragrunt/**"
+      - ".github/workflows/tf_apply_staging.yml"
+
+env:
+  TERRAFORM_VERSION: 1.0.3
+  TERRAGRUNT_VERSION: 0.38.4
+  TF_VAR_api_auth_token: ${{ secrets.STAGING_API_AUTH_TOKEN }}
+  TF_VAR_aws_org_id: ${{ secrets.AWS_ORG_ID }}
```

Comment:
We'll need a `Staging` specific AWS_ORG_ID secret since the two accounts are in diff orgs right now.

Reply:
Nice catch. I meant to use `STAGING_AWS_ORG_ID`